builtin/providers/cidr/datasource_cidr_subnet.go

```diff
@@ -0,0 +1,108 @@
+package cidr
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceSubnet() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceSubnetRead,
+
+		Schema: map[string]*schema.Schema{
+			"cidr_block": &schema.Schema{
+				Type:        schema.TypeString,
+				Description: "The CIDR Block for the entire network (aka supernet)",
```

Comment:
*block* (lowercase)

Reply:
fixed in https://github.com/moofish32/terraform-provider-cidr/pull/2